README.md

```diff
@@ -6,38 +6,39 @@ This is an open-ended pair lab! It is up to you and your teammate to come up wit
 
```

Comment:
Should we add any language here about how they should approach pair programming (I.E. a quick reminder of driver / navigator roles and whether or not they should switch off) or should we let them figure this out for themselves?